### PR TITLE
fix: fixture generation

### PR DESCRIPTION
**Pull request by william-brooks**

### CODE REVIEW

File: solana/program_v2/package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "mocha": "^10.0.0",
     "prettier": "^2.7.1",
     "ts-mocha": "^10.0.0",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "@solana/spl-token": "^0.3.5"
```
Comment (rado0x54) [inline solana/program_v2/package.json:39]: This should not be in the root package, right? :)

Comment (william-brooks) [reply solana/program_v2/package.json:39]: the `scripts` (to generate the fixtures) were moved to the root... I added a "mint" generation for us to use for testing... hence needing the `spl-token` lib

Comment (rado0x54) [reply solana/program_v2/package.json:39]: Shouldn't the fixture generation be in the tests package? As mentioned before I'd like to root to be as this as possible.

**Review by DiosAkhilleus — APPROVED** [review]
Looks great!

**Pull request merged**